docs/development.md

```diff
@@ -73,6 +73,14 @@ host:~/Projects$ cd voyages
 host:~/Projects/voyages$ git remote add upstream https://github.com/rice-crc/voyages.git
 ```
 
+Note: if you are cloning for development, it is likely that you will be working
+on different branches at different times. It is very important to create
+separate DB and Solr Cores for each branch since the schema/data could be
+different. In the localsettings.py.dist example file, there is support for
```

Comment:
Change to "localsettings-local.py.default".

Reply:
Good catch

Reply:
Fixed in 2347968